src/lib/y2storage/boot_requirements_strategies/base.rb

```diff
@@ -116,6 +121,14 @@ def boot_partition_needed?
         false
       end
 
+      def too_small_boot?
+        filesystem = devicegraph.filesystems.find { |f| f.mount_point == "/boot" }
+        return false unless filesystem
+
+        available_size = filesystem.blk_devices.map(&:size).reduce(:+)
+        return available_size < DiskSize.MiB(100) # for number see below minimal size for boot volume
```

Comment:
Well, 100MiB is the minimal size the proposal will use as starting point, which doesn't imply is the minimal size to contain a kernel .

Reply:
I think current kernel require around 67MiB, but usually you need to have here also some boot code. and with kernel multiversioning it takes more. And kernel is still growing, so for me 100 MiB is reasonable size.

Reply:
I think that, if you rebase on top of #534 you could use this here:
```ruby
def too_small_boot?
  #... [stripped] ...
  available_size < min_boot_size
end

def min_boot_size
  @min_boot_size ||= VolumeSpecificationBuilder.new.for("/boot").min_size
end
```

Instead of repeating the value here.